### PR TITLE
Remove unused rtcspec.

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java [hunk 1]
```diff
@@ -139,22 +137,13 @@ public NodeSpec getNodeSpec() {
...
-
-    /**
      * Returns the number of ports needed by this service.
      *
      * @return The number of ports.
      */
     @Override
     public int getPortCount() {
-        return 5;
+        return 3;
```
Comment (toregge) [inline config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java]: Port number changes might cause issues during version upgrade, where some nodes run with old config and some nodes run with new config.

File: config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java [hunk 2]
```diff
@@ -208,7 +184,7 @@ public int getDispatchPort() {
     }
 
     public int getHttpPort() {
-        return getRelativePort(4);
+        return getRelativePort(2);
```
Comment (toregge) [inline config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java]: Port number changes might cause issues during version upgrade, where some nodes run with old config and some nodes run with new config.  e.g. health check trying to use wrong port will see node as down.

File: config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java [hunk 3]
```diff
@@ -166,19 +155,6 @@ public int getRpcPort() {
         return getRelativePort(0);
     }
 
-    protected int getSlimeMessagingPort() {
```
Comment (toregge) [inline config-model/src/main/java/com/yahoo/vespa/model/search/SearchNode.java]: Removal of slime messaging port should be reviewed by @havardpe.

**Comment by baldersheim** [comment]
@hmusum Might this cause issues ?
- 2 unused ports are removed form searchnode. 
- port count is reduced to from 5 -> 3.
- health/metrics port is moved from offset 4 to 2.

**Comment by hmusum** [comment]
Yes, it might cause issues as Tor says. We have usually ended up doing `tag("unused")` on ports we no longer use due to this.

**Review by havardpe — COMMENTED** [review]
Feel free to remove the slime messaging port. I guess we will use the health/metric port for this anyways.

**Comment by baldersheim** [comment]
So conclusion is to not move health port now, but in step 2 move it, and then in step 3 reduce number of ports.

**Pull request merged**